django/http/multipartparser.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, META, input_data, upload_handlers, encoding=None):
         ctypes, opts = parse_header(content_type.encode('ascii'))
         boundary = opts.get('boundary')
```

Comment:
We would fallback to an empty `bytes` string as well

```suggestion
        boundary = opts.get('boundary', b'')
```